README.md

```diff
@@ -23,6 +23,25 @@ A `piskel-root` utilty is also installed at `node_modules/.bin/piskel-root` that
 
 Note: To run local integration tests you should install CasperJS 1.0.2 (not included as a dependency in this repo) and make sure it has access to PhantomJS 1.9.2 (downloaded to node_modules/.bin on `npm install` but not necessarily in your PATH).
 
+## Contributing
+**More on contributing can be found in the [main Piskel repo](https://github.com/piskelapp/piskel/wiki#contributing)**
+
+### Prerequisite
+To build Piskel, you need to :
+- install node and grunt-cli `npm install grunt-cli -g`.
```

Comment:
that command doesn't also install node, right? node should probably be it's own line? What version of node should be installed?

Reply:
I am looking at the contributing guide: [https://github.com/piskelapp/piskel/wiki/Development-Environment](https://github.com/piskelapp/piskel/wiki/Development-Environment) and it just says to install node. Does not say which version, i can link them to the nodejs.org website